Fixed IndexError list index out of range for Squeeze

abhinav231-valisetti opened the pull request:
### Details:
 - *I have added a check to ensure the list is not empty before accessing its elements, addressing the IndexError issue in the Squeeze operation.*

Issue - 
https://github.com/openvinotoolkit/openvino/issues/22078

rkazants commented on src/bindings/python/src/openvino/frontend/tensorflow/node_decoder.py:
```diff
@@ -36,7 +36,11 @@ def tf_attr_to_numpy(attr):
+        if fields and len(fields) > 0 and len(fields[0]) > 1:
+            return list(fields[0][1])
+        else:
+            return
```
should we return `None` here because it expects the returned value from the function?

rkazants reviewed (commented):
@abhinav231-valisetti, please update https://github.com/openvinotoolkit/openvino/blob/master/tests/model_hub_tests/tf_hub_tests/precommit_models with a model for which you did this fix so your fix will be validated in the pre-commit.

p-wysocki commented:
build_jenkins

rkazants commented on tests/model_hub_tests/tf_hub_tests/nightly_models:
```diff
@@ -65,12 +65,12 @@ biggan-deep-256,https://www.kaggle.com/models/deepmind/biggan/frameworks/tensorF
 imagenet/mobilenet_v2_050_224/classification,https://www.kaggle.com/models/google/mobilenet-v2/frameworks/tensorFlow2/variations/050-224-classification/versions/2
 mobilebert/uncased_L-24_H-128_B-512_A-4_F-4_OPT/squadv1,https://www.kaggle.com/models/google/mobilebert/frameworks/tensorFlow1/variations/uncased-l-24-h-128-b-512-a-4-f-4-opt-squadv1/versions/1
-delf,https://www.kaggle.com/models/google/delf/frameworks/tensorFlow1/variations/delf/versions/1,xfail,129232 IndexError list index out of range for Squeeze
+delf,https://www.kaggle.com/models/google/delf/frameworks/tensorFlow1/variations/delf/versions/1,129232 IndexError list index out of range for Squeeze
```
```suggestion
delf,https://www.kaggle.com/models/google/delf/frameworks/tensorFlow1/variations/delf/versions/1
```

rkazants replied:
we just need to remain only two columns

abhinav231-valisetti commented:
Hi @rkazants I have performed the following changes in my recent commit,
1) Added a Model in the precommit_models for precommit validation.
   `mmt/data_cc,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/data-cc/versions/1`
2) Removed skip and xfail marrks for the issue (129232 IndexError list index out of range for Squeeze) in nighty_models.
3) Added explicilty return None in node_decoder.py

*** _Below model which i used to fix the current issue initially is giving me an error while loading the modal using tensowflow_hub. So i have used another model from the nighty models to move forward._

rkazants commented on tests/model_hub_tests/tf_hub_tests/nightly_models:
```diff
@@ -65,12 +65,12 @@ biggan-deep-256,https://www.kaggle.com/models/deepmind/biggan/frameworks/tensorF
 centernet/hourglass_512x512_kpts,https://www.kaggle.com/models/tensorflow/centernet-hourglass/frameworks/tensorFlow2/variations/512x512-kpts/versions/1,skip,129153 TimeoutError or killed with a signal 11
 bert_cased_L-12_H-768_A-12,https://www.kaggle.com/models/google/bert/frameworks/tensorFlow1/variations/cased-l-12-h-768-a-12/versions/1,skip,129153 TimeoutError or killed with a signal 11
-biggan-512,https://www.kaggle.com/models/deepmind/biggan/frameworks/tensorFlow1/variations/512/versions/2,skip,129232 IndexError list index out of range for Squeeze
+biggan-512,https://www.kaggle.com/models/deepmind/biggan/frameworks/tensorFlow1/variations/512/versions/2,129232 IndexError list index out of range for Squeeze
```
```suggestion
biggan-512,https://www.kaggle.com/models/deepmind/biggan/frameworks/tensorFlow1/variations/512/versions/2
```

rkazants commented on tests/model_hub_tests/tf_hub_tests/nightly_models:
```diff
@@ -87,7 +87,7 @@ imagenet/efficientnet_v2_imagenet21k_ft1k_m/feature_vector,https://www.kaggle.co
 bit/m-r50x1,https://www.kaggle.com/models/google/bit/frameworks/tensorFlow2/variations/m-r50x1/versions/1
 progan-128,https://www.kaggle.com/models/google/progan-128/frameworks/tensorFlow1/variations/progan-128/versions/1
-biggan-256,https://www.kaggle.com/models/deepmind/biggan/frameworks/tensorFlow1/variations/256/versions/2,skip,129232 IndexError list index out of range for Squeeze
+biggan-256,https://www.kaggle.com/models/deepmind/biggan/frameworks/tensorFlow1/variations/256/versions/2,129232 IndexError list index out of range for Squeeze
```
```suggestion
biggan-256,https://www.kaggle.com/models/deepmind/biggan/frameworks/tensorFlow1/variations/256/versions/2
```

rkazants commented on tests/model_hub_tests/tf_hub_tests/nightly_models:
```diff
@@ -159,7 +159,7 @@ faster_rcnn/inception_resnet_v2_1024x1024,https://www.kaggle.com/models/tensorfl
 imagenet/resnet_v1_50/classification,https://www.kaggle.com/models/google/resnet-v1/frameworks/tensorFlow2/variations/50-classification/versions/2
 movinet/a5/base/kinetics-600/classification,https://www.kaggle.com/models/google/movinet/frameworks/tensorFlow2/variations/a5-base-kinetics-600-classification/versions/3
-biggan-128,https://www.kaggle.com/models/deepmind/biggan/frameworks/tensorFlow1/variations/128/versions/2,skip,129232 IndexError list index out of range for Squeeze
+biggan-128,https://www.kaggle.com/models/deepmind/biggan/frameworks/tensorFlow1/variations/128/versions/2,129232 IndexError list index out of range for Squeeze
```
```suggestion
biggan-128,https://www.kaggle.com/models/deepmind/biggan/frameworks/tensorFlow1/variations/128/versions/2
```

rkazants commented on tests/model_hub_tests/tf_hub_tests/nightly_models:
```diff
@@ -394,7 +394,7 @@ german-tacotron2,https://www.kaggle.com/models/myusufs/german-tacotron2/framewor
 gtr/gtr-large,https://www.kaggle.com/models/google/gtr/frameworks/tensorFlow2/variations/gtr-large/versions/1
 AraBERT,https://www.kaggle.com/models/kaggle/arabert/frameworks/tensorFlow2/variations/arabert/versions/1
-trillsson1,https://www.kaggle.com/models/google/trillsson/frameworks/tensorFlow2/variations/1/versions/1,skip,129232 IndexError list index out of range for Squeeze
+trillsson1,https://www.kaggle.com/models/google/trillsson/frameworks/tensorFlow2/variations/1/versions/1,129232 IndexError list index out of range for Squeeze
```
```suggestion
trillsson1,https://www.kaggle.com/models/google/trillsson/frameworks/tensorFlow2/variations/1/versions/1
```

rkazants commented on tests/model_hub_tests/tf_hub_tests/nightly_models:
```diff
@@ -423,7 +423,7 @@ wav2vec2-xlsr-53,https://www.kaggle.com/models/kaggle/wav2vec2/frameworks/tensor
 movinet/a5/stream/kinetics-600/classification,https://www.kaggle.com/models/google/movinet/frameworks/tensorFlow2/variations/a5-stream-kinetics-600-classification/versions/2
 vit_r26_s32_lightaug_fe,https://www.kaggle.com/models/spsayakpaul/vision-transformer/frameworks/tensorFlow2/variations/vit-r26-s32-lightaug-fe/versions/1
-mmt/architecture_image-q-24,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/architecture-image-q-24/versions/1,skip,129232 IndexError list index out of range for Squeeze
+mmt/architecture_image-q-24,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/architecture-image-q-24/versions/1,129232 IndexError list index out of range for Squeeze
```
```suggestion
mmt/architecture_image-q-24,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/architecture-image-q-24/versions/1
```

rkazants commented on tests/model_hub_tests/tf_hub_tests/nightly_models:
```diff
@@ -470,7 +470,7 @@ sup-rotation-100,https://www.kaggle.com/models/vtab/sup-rotation-100/frameworks/
 imagenet/mobilenet_v2_075_128/classification,https://www.kaggle.com/models/google/mobilenet-v2/frameworks/tensorFlow2/variations/075-128-classification/versions/2
 efficientdet/lite3x/feature-vector,https://www.kaggle.com/models/tensorflow/efficientdet/frameworks/tensorFlow2/variations/lite3x-feature-vector/versions/1
-mmt/baseline_baseline,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/baseline-baseline/versions/1,skip,129232 IndexError list index out of range for Squeeze
+mmt/baseline_baseline,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/baseline-baseline/versions/1,129232 IndexError list index out of range for Squeeze
```
```suggestion
mmt/baseline_baseline,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/baseline-baseline/versions/1
```

rkazants commented on tests/model_hub_tests/tf_hub_tests/nightly_models:
```diff
@@ -499,14 +499,14 @@ vgg19-block3-conv2-unpooling-encoder,https://www.kaggle.com/models/emilutz/vgg19
 remote_sensing/uc_merced-resnet50,https://www.kaggle.com/models/google/resnet50/frameworks/tensorFlow1/variations/remote-sensing-uc-merced-resnet50/versions/1
 mixer_b16_i1k_fe,https://www.kaggle.com/models/spsayakpaul/mlp-mixer/frameworks/tensorFlow2/variations/mixer-b16-i1k-fe/versions/1
-mmt/loss_single-modality-contrastive1024,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/loss-single-modality-contrastive1024/versions/1,skip,129232 IndexError list index out of range for Squeeze
+mmt/loss_single-modality-contrastive1024,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/loss-single-modality-contrastive1024/versions/1,129232 IndexError list index out of range for Squeeze
```
```suggestion
mmt/loss_single-modality-contrastive1024,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/loss-single-modality-contrastive1024/versions/1
```

rkazants commented on tests/model_hub_tests/tf_hub_tests/nightly_models:
```diff
@@ -499,14 +499,14 @@ vgg19-block3-conv2-unpooling-encoder,https://www.kaggle.com/models/emilutz/vgg19
 mixer_b16_sam_fe,https://www.kaggle.com/models/spsayakpaul/mlp-mixer/frameworks/tensorFlow2/variations/mixer-b16-sam-fe/versions/1
 logit_reconstruction/robust,https://www.kaggle.com/models/google/logit-reconstruction/frameworks/tensorFlow1/variations/robust/versions/1
-mmt/baseline_baseline-no-bert-transfer,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/baseline-baseline-no-bert-transfer/versions/1,skip,129232 IndexError list index out of range for Squeeze
+mmt/baseline_baseline-no-bert-transfer,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/baseline-baseline-no-bert-transfer/versions/1,129232 IndexError list index out of range for Squeeze
```
```suggestion
mmt/baseline_baseline-no-bert-transfer,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/baseline-baseline-no-bert-transfer/versions/1
```

rkazants commented on tests/model_hub_tests/tf_hub_tests/nightly_models:
```diff
@@ -518,7 +518,7 @@ spiral/default-wgangp-celebahq64-gen-19steps/agent8,https://www.kaggle.com/model
 HRNet/ade20k-hrnetv2-w48,https://www.kaggle.com/models/google/hrnet/frameworks/tensorFlow2/variations/hrnet-ade20k-hrnetv2-w48/versions/1,skip,120721 AssertionError No signatures for a model
 wae-gan,https://www.kaggle.com/models/vtab/wae-gan/frameworks/tensorFlow1/variations/wae-gan/versions/1
-mmt/architecture_image-q-12,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/architecture-image-q-12/versions/1,skip,129232 IndexError list index out of range for Squeeze
+mmt/architecture_image-q-12,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/architecture-image-q-12/versions/1,129232 IndexError list index out of range for Squeeze
```
```suggestion
mmt/architecture_image-q-12,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/architecture-image-q-12/versions/1
```

rkazants commented on tests/model_hub_tests/tf_hub_tests/nightly_models:
```diff
@@ -547,14 +547,14 @@ remote_sensing/eurosat-ms-resnet50,https://www.kaggle.com/models/google/resnet50
 convnext_xlarge_21k_1k_384_fe,https://www.kaggle.com/models/spsayakpaul/convnext/frameworks/tensorFlow2/variations/xlarge-21k-1k-384-fe/versions/1
 mobilevit_xxs_1k_256_fe,https://www.kaggle.com/models/kaggle/mobilevit/frameworks/tensorFlow2/variations/xxs-1k-256-fe/versions/1
-mmt/data_cc,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/data-cc/versions/1,skip,129232 IndexError list index out of range for Squeeze
+mmt/data_cc,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/data-cc/versions/1,129232 IndexError list index out of range for Squeeze
```
```suggestion
mmt/data_cc,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/data-cc/versions/1
```

rkazants commented on tests/model_hub_tests/tf_hub_tests/nightly_models:
```diff
@@ -547,14 +547,14 @@ remote_sensing/eurosat-ms-resnet50,https://www.kaggle.com/models/google/resnet50
 mixer_b16_i21k_fe,https://www.kaggle.com/models/spsayakpaul/mlp-mixer/frameworks/tensorFlow2/variations/mixer-b16-i21k-fe/versions/1,skip,128695 Inference results mismatch
-nonsemantic-speech-benchmark/trillsson1,https://www.kaggle.com/models/google/trillsson/frameworks/tensorFlow2/variations/nonsemantic-speech-benchmark-trillsson1/versions/1,skip,129232 IndexError list index out of range for Squeeze
-trillsson5,https://www.kaggle.com/models/google/trillsson/frameworks/tensorFlow2/variations/5/versions/1,skip,129232 IndexError list index out of range for Squeeze
+nonsemantic-speech-benchmark/trillsson1,https://www.kaggle.com/models/google/trillsson/frameworks/tensorFlow2/variations/nonsemantic-speech-benchmark-trillsson1/versions/1,129232 IndexError list index out of range for Squeeze
+trillsson5,https://www.kaggle.com/models/google/trillsson/frameworks/tensorFlow2/variations/5/versions/1,129232 IndexError list index out of range for Squeeze
```
```suggestion
nonsemantic-speech-benchmark/trillsson1,https://www.kaggle.com/models/google/trillsson/frameworks/tensorFlow2/variations/nonsemantic-speech-benchmark-trillsson1/versions/1
trillsson5,https://www.kaggle.com/models/google/trillsson/frameworks/tensorFlow2/variations/5/versions/1
```

rkazants commented on tests/model_hub_tests/tf_hub_tests/nightly_models:
```diff
@@ -589,7 +589,7 @@ sentence-t5/st5-11b,https://www.kaggle.com/models/google/sentence-t5/frameworks/
 swin_base_patch4_window7_224_fe,https://www.kaggle.com/models/spsayakpaul/swin/frameworks/tensorFlow2/variations/base-patch4-window7-224-fe/versions/1
 small_bert/bert_uncased_L-8_H-128_A-2,https://www.kaggle.com/models/google/bert/frameworks/tensorFlow1/variations/bert-uncased-l-8-h-128-a-2/versions/2,skip,129153 TimeoutError or killed with a signal 11
-mmt/data_mscoco,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/data-mscoco/versions/1,skip,129232 IndexError list index out of range for Squeeze
+mmt/data_mscoco,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/data-mscoco/versions/1,129232 IndexError list index out of range for Squeeze
```
```suggestion
mmt/data_mscoco,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/data-mscoco/versions/1
```

rkazants commented on tests/model_hub_tests/tf_hub_tests/nightly_models:
```diff
@@ -606,7 +606,7 @@ swin_small_patch244_window877_kinetics400_1k,https://www.kaggle.com/models/kaggl
 HRNet/msegpcontext-hrnetv2-w48,https://www.kaggle.com/models/google/hrnet/frameworks/tensorFlow2/variations/hrnet-msegpcontext-hrnetv2-w48/versions/1,skip,120721 AssertionError No signatures for a model
 convnext_large_21k_1k_224_fe,https://www.kaggle.com/models/spsayakpaul/convnext/frameworks/tensorFlow2/variations/large-21k-1k-224-fe/versions/1
-mmt/baseline-ft_baseline-no-bert-transfer,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/baseline-ft-baseline-no-bert-transfer/versions/1,skip,129232 IndexError list index out of range for Squeeze
+mmt/baseline-ft_baseline-no-bert-transfer,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/baseline-ft-baseline-no-bert-transfer/versions/1,129232 IndexError list index out of range for Squeeze
```
```suggestion
mmt/baseline-ft_baseline-no-bert-transfer,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/baseline-ft-baseline-no-bert-transfer/versions/1
```

rkazants commented on tests/model_hub_tests/tf_hub_tests/nightly_models:
```diff
@@ -622,20 +622,20 @@ experts/bit/r50x1/in21k/abstraction,https://www.kaggle.com/models/google/experts
 imagenet/efficientnet_v2_imagenet21k_b1/classification,https://www.kaggle.com/models/google/efficientnet-v2/frameworks/tensorFlow2/variations/imagenet21k-b1-classification/versions/1
-mmt/data_sbu,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/data-sbu/versions/1,skip,129232 IndexError list index out of range for Squeeze
-mmt/data_combined-instance,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/data-combined-instance/versions/1,skip,129232 IndexError list index out of range for Squeeze
+mmt/data_sbu,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/data-sbu/versions/1,129232 IndexError list index out of range for Squeeze
+mmt/data_combined-instance,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/data-combined-instance/versions/1,129232 IndexError list index out of range for Squeeze
```
```suggestion
mmt/data_sbu,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/data-sbu/versions/1
mmt/data_combined-instance,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/data-combined-instance/versions/1
```

rkazants commented on tests/model_hub_tests/tf_hub_tests/nightly_models:
```diff
@@ -622,20 +622,20 @@ experts/bit/r50x1/in21k/abstraction,https://www.kaggle.com/models/google/experts
 imagenet/mobilenet_v1_075_160/quantops/classification,https://www.kaggle.com/models/google/mobilenet-v1/frameworks/tensorFlow1/variations/075-160-quantops-classification/versions/2,skip,128695 Inference results mismatch
 convnext_large_21k_1k_224,https://www.kaggle.com/models/spsayakpaul/convnext/frameworks/tensorFlow2/variations/large-21k-1k-224/versions/1
-mmt/architecture_vilbert-4block,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/architecture-vilbert-4block/versions/1,skip,129232 IndexError list index out of range for Squeeze
+mmt/architecture_vilbert-4block,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/architecture-vilbert-4block/versions/1,129232 IndexError list index out of range for Squeeze
```
```suggestion
mmt/architecture_vilbert-4block,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/architecture-vilbert-4block/versions/1
```

rkazants commented on tests/model_hub_tests/tf_hub_tests/nightly_models:
```diff
@@ -622,20 +622,20 @@ experts/bit/r50x1/in21k/abstraction,https://www.kaggle.com/models/google/experts
+mmt/architecture_vilbert-4block,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/architecture-vilbert-4block/versions/1,129232 IndexError list index out of range for Squeeze
 imagenet/mobilenet_v1_050_160/quantops/classification,https://www.kaggle.com/models/google/mobilenet-v1/frameworks/tensorFlow1/variations/050-160-quantops-classification/versions/2,skip,128695 Inference results mismatch
-mmt/architecture-ft_language-q-12,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/architecture-ft-language-q-12/versions/1,skip,129232 IndexError list index out of range for Squeeze
+mmt/architecture-ft_language-q-12,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/architecture-ft-language-q-12/versions/1,129232 IndexError list index out of range for Squeeze
```
```suggestion
mmt/architecture-ft_language-q-12,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/architecture-ft-language-q-12/versions/1
```

rkazants commented on tests/model_hub_tests/tf_hub_tests/nightly_models:
```diff
@@ -622,20 +622,20 @@ experts/bit/r50x1/in21k/abstraction,https://www.kaggle.com/models/google/experts
 bit_resnet152x2_224_classification,https://www.kaggle.com/models/spsayakpaul/bit-resnet/frameworks/tensorFlow2/variations/bit-resnet152x2-224-classification/versions/1
 experts/bit/r50x1/in21k/relation,https://www.kaggle.com/models/google/experts-bit/frameworks/tensorFlow2/variations/r50x1-in21k-relation/versions/1
-mmt/architecture_single-modality,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/architecture-single-modality/versions/1,skip,129232 IndexError list index out of range for Squeeze
+mmt/architecture_single-modality,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/architecture-single-modality/versions/1,129232 IndexError list index out of range for Squeeze
```
```suggestion
mmt/architecture_single-modality,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/architecture-single-modality/versions/1
```

rkazants commented on tests/model_hub_tests/tf_hub_tests/nightly_models:
```diff
@@ -649,7 +649,7 @@ deit_base_patch16_384,https://www.kaggle.com/models/spsayakpaul/deit/frameworks/
 llr-pretrain-adv/linear,https://www.kaggle.com/models/deepmind/llr-pretrain-adv/frameworks/tensorFlow1/variations/linear/versions/1
 edgetpu/vision/mobilenet-edgetpu-v2-feature-vector/m,https://www.kaggle.com/models/google/mobilenet-edgetpu-v2/frameworks/tensorFlow2/variations/feature-vector-m/versions/2
-mmt/architecture_vilbert-2block,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/architecture-vilbert-2block/versions/1,skip,129232 IndexError list index out of range for Squeeze
+mmt/architecture_vilbert-2block,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/architecture-vilbert-2block/versions/1,129232 IndexError list index out of range for Squeeze
```
```suggestion
mmt/architecture_vilbert-2block,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/architecture-vilbert-2block/versions/1
```

rkazants commented on tests/model_hub_tests/tf_hub_tests/nightly_models:
```diff
@@ -668,9 +668,9 @@ mixer_b32_sam_fe,https://www.kaggle.com/models/spsayakpaul/mlp-mixer/frameworks/
 tiny_video_net/mobile_1,https://www.kaggle.com/models/google/tiny-video-net/frameworks/tensorFlow1/variations/mobile-1/versions/1,skip,128817 Model references undeclared parameters
 vgg19-block5-conv2-unpooling-encoder,https://www.kaggle.com/models/emilutz/vgg19-unpooling-encoder-decoder/frameworks/tensorFlow2/variations/vgg19-block5-conv2-unpooling-encoder/versions/1,skip,128695 Inference results mismatch
-mmt/data-ft_sbu,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/data-ft-sbu/versions/1,skip,129232 IndexError list index out of range for Squeeze
+mmt/data-ft_sbu,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/data-ft-sbu/versions/1,129232 IndexError list index out of range for Squeeze
```
```suggestion
mmt/data-ft_sbu,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/data-ft-sbu/versions/1
```

rkazants commented on tests/model_hub_tests/tf_hub_tests/nightly_models:
```diff
@@ -668,9 +668,9 @@ mixer_b32_sam_fe,https://www.kaggle.com/models/spsayakpaul/mlp-mixer/frameworks/
+mmt/data-ft_sbu,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/data-ft-sbu/versions/1,129232 IndexError list index out of range for Squeeze
 swin_s3_base_224,https://www.kaggle.com/models/spsayakpaul/swin/frameworks/tensorFlow2/variations/s3-base-224/versions/1
-mmt/data_cc-with-bert,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/data-cc-with-bert/versions/1,skip,129232 IndexError list index out of range for Squeeze
+mmt/data_cc-with-bert,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/data-cc-with-bert/versions/1,129232 IndexError list index out of range for Squeeze
```
```suggestion
mmt/data_cc-with-bert,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/data-cc-with-bert/versions/1
```

rkazants commented on tests/model_hub_tests/tf_hub_tests/precommit_models:
```diff
@@ -1,3 +1,4 @@
+mmt/data_cc,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/data-cc/versions/1
```
```suggestion
mmt/data_cc,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/data-cc/versions/1
delf,https://www.kaggle.com/models/google/delf/frameworks/tensorFlow1/variations/delf/versions/1
```

rkazants replied:
let us experiment with delf

rkazants commented:
build_jenkins

rkazants commented on tests/model_hub_tests/tf_hub_tests/precommit_models:
```diff
@@ -1,3 +1,5 @@
+mmt/data_cc,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/data-cc/versions/1
+delf,https://www.kaggle.com/models/google/delf/frameworks/tensorFlow1/variations/delf/versions/1
```
```suggestion
mmt/data_cc,https://www.kaggle.com/models/deepmind/mmt/frameworks/tensorFlow1/variations/data-cc/versions/1,skip,CPU plug-in doesn't support Squeeze operation with dynamic rank Operation name object_classification_layer/Squeeze
```

rkazants commented:
build_jenkins

rkazants commented:
build_jenkins
